### DISCUSSION TITLE
Dungeon and Raid Progress Tracker

### DISCUSSION THREAD

**Discussion by Xevius95** [blish-hud/Blish-HUD#699] [Module Ideas]
Hello, 
a small tool for graphically displaying the dungeon frequenters or the raid kills would be a useful addition :)
While such tools already exist (Arc Clears and TaCo) it would be helpful to have both in one place! :D

![Bild_2022-05-30_114219505](https://user-images.githubusercontent.com/92651197/170965116-17bae875-31a1-49e0-88a5-2fb450a51906.png)
![Bild_2022-05-30_114304820](https://user-images.githubusercontent.com/92651197/170965265-cbadd9fa-20d1-4e42-a4d6-5ecd7c4a4104.png)

**Comment by Taschenbuch** [comment]
Is implemented by RAID CLEARS module and CM TRACKER module
